### PR TITLE
Fe jy news task4

### PR DESCRIPTION
**Pull request by jamesyuill**

### CODE REVIEW

File: src/utils/api.js
Comment (NRMSMITH) [inline src/utils/api.js]: Consider making a baseURL that you will be able to access, as if your api needed to change address, you would only need to do this in one area. Also, I would suggest using axios as it converts the data to json which is much easier for readability - but perhaps this one is fine as is! Here are the notes: https://notes.northcoders.com/courses/js-front-end/axios

File: src/components/Content.jsx
Comment (NRMSMITH) [inline src/components/Content.jsx:8]: some loading state would be excellent - just in case the internet connection is slow. It will enhance the user experience 😄

File: src/pages/ArticleList.jsx
Comment (NRMSMITH) [inline src/pages/ArticleList.jsx]: does this need to be a div? Is there anything that is a bit more semantic that you could use?

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
A few changes to make in the comments :) I really like that your design is alreay responsive! well done!

Please resubmit once done 😄

File: src/components/Content.jsx
Comment (B7Bprog) [inline src/components/Content.jsx]: Please remove this log.

File: src/components/Content.jsx
Comment (B7Bprog) [inline src/components/Content.jsx:12]: The 'isLoading' state could be set to true here just before the API request, so then it would indicate loading even later when you want to fetch the articles again on the page.

File: src/utils/api.js
Comment (B7Bprog) [inline src/utils/api.js]: Please use axios here so you demonstrate that you are able to use that library. It will make things simpler later as well.

File: src/App.css
Comment (B7Bprog) [inline src/App.css:56]: Currently when I open the article list, I can only see the images. All the text (like title) are white so they are not visible. A colour for that might need to be specified somewhere in your .css file.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey James, great work so far, only a few minor things left to do. Please see my comments, refactor your code, then send us this link again on Slack after a commit into this PR.

File: src/components/ArticleCard.jsx
Comment (itskatherine) [inline src/components/ArticleCard.jsx:9]: Later on (Ticket 10) we're going to be adding functionality to sort these articles by things such as the date they were written or comment count, it would be good for these things to be visible on the article cards when that happens to give an indication to the user it's worked. Could either add it here now or later, I'll leave it up to you 🙂

File: src/components/Content.jsx
Comment (itskatherine) [inline src/components/Content.jsx:34]: This header's a lil confusing when there's no content on the "/" path 🤔

File: src/utils/api.js
Comment (itskatherine) [inline src/utils/api.js:3]: Nice use of axios ⭐ 🏆 🎉

**Review by itskatherine — COMMENTED** [review]
Hullooo James there's a few minor things to look at but it's good to merge once you've had a look at my comments! 👍

File: src/components/Content.jsx
Comment (jamesyuill) [reply src/components/Content.jsx:34]: There will be, I've allowed for the home page to be top 5 most recent articles, so home "/" will always have most recent articles on.

File: src/components/ArticleCard.jsx
Comment (jamesyuill) [reply src/components/ArticleCard.jsx:9]: Cool! I understand. I can add them in now. no problem. just wanted to keep the article preview minimal for now compared to the full article, but I see why we'd need that info on the big article list

**Pull request merged**
